### ISSUE TITLE
Add the siad version to the startup message

### ISSUE THREAD

**Issue by DavidVorick** [NebulousLabs/Sia#2293]
One of the first things we always check in every issue that a user reports is the version of their Sia daemon, because you can't usually figure that out just from the logs.

If we add a message that includes the version number in the startup output, and perhaps in the startup message for `persist.NewLogger` as well. While the output is not strictly necessary, it will help a lot when debugging and allow us to skip the step of asking what version someone is using, and also remove any doubt that they are incorrect (as we sometimes do have users who thought they were running the latest version but siad is actually running a previous version still).

**Comment by eddiewang** [comment]
Console looks like this:
```
Sia Daemon v1.3.0
Loading...
(0/8) Loading siad...
(1/8) Loading gateway...
(2/8) Loading consensus...
(3/8) Loading explorer...
(4/8) Loading transaction pool...
(5/8) Loading wallet...
(6/8) Loading miner...
(7/8) Loading host...
(8/8) Loading renter...
Finished loading in 0.575467625 seconds
```
Log looks like this:
```
2017/09/05 22:55:21.337637 persist.go:403: STARTUP: Logging has started. Siad Version 1.3.0
```

**Comment by mtlynch** [comment]
This was fixed in #2294.
